aiida_cp2k/calculations/datatype_helpers.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+# Copyright (c), The AiiDA-CP2K authors.                                      #
+# SPDX-License-Identifier: MIT                                                #
+# AiiDA-CP2K is hosted on GitHub at https://github.com/aiidateam/aiida-cp2k   #
+# For further information on the license, see the LICENSE.txt file.           #
+###############################################################################
+"""AiiDA-CP2K Gaussian Datatype Helpers."""
+
+import io
+from collections.abc import Sequence
+
+from aiida.common import InputValidationError
+from aiida.plugins import DataFactory
+
+
+def _unpack(adict):
```

Comment:
Can you make this function to be a member of `Cp2kInput` class?

Reply:
I could, yes, but it has nothing to do with the `Cp2kInput` class and it is not even used on data inside `Cp2kInput`.

Reply:
Can you elaborate?

Reply:
Honestly, I don't remember what I had in mind. And I agree with you.